src/components/nodes/pool/pool.vue

```diff
@@ -1,23 +1,51 @@
 <template>
-  <div/>
+  <crown-config
+    :highlighted="highlighted"
+    :paper="paper"
+    :graph="graph"
+    :shape="shape"
+    :node="node"
+    :nodeRegistry="nodeRegistry"
+    :moddle="moddle"
+    :collaboration="collaboration"
+    :process-node="processNode"
+    :plane-elements="planeElements"
+    :is-rendering="isRendering"
+    @remove-node="$emit('remove-node', $event)"
+    @add-node="$emit('add-node', $event)"
+    @save-state="$emit('save-state', $event)"
+  >
```

Comment:
Pool has allowSetNodePosition which is in the crown component.
Does `element.component.allowSetNodePosition` in this file still work with the new crown component?

Reply:
Yup, that still seems to work as expected. Did you notice any problems with it?